proto/eth/service/key_management.proto

```diff
@@ -167,6 +167,19 @@ service KeyManagement {
       body: "*"
     };
   }
+
+  // GetGasLimit returns the execution gas limit for an individual validator.
+  //
+  // HTTP response status codes:
+  //  - 400: Bad request
+  //  - 401: Unauthorized
+  //  - 403: Forbidden
+  //  - 500: Validator internal error
+  rpc GetGasLimit(PubkeyRequest) returns (GetGasLimitResponse) {
+    option (google.api.http) = {
```

Comment:
you need to regenerate the proto using `./hack/update-go-pbs.sh` and updating the generated files associated to the proto. the proto itself doesn't contain the implementation of the function.

Reply:
Thanks. Ran update-go-pbs.sh and only included key_management.pb* in the PR.